docs/tutorial_human_popgen.md

```diff
@@ -0,0 +1,515 @@
+-name 'projectX_screening20200727' \
+--input 'screening20200727.tsv' \
+--fasta '../Reference/genome/hs37d5.fa' \
+--bwa_index '../Reference/genome/hs37d5/' \
+--fasta_index '../Reference/genome/hs37d5.fa.fai' \
+--seq_dict '../Reference/genome/hs37d5.dict' \
+--outdir './results/' \
+- w './work/' \
+--complexity_filter_poly_g \
+--preserve5p \
+--bwaalnn 0.01 \
+--run_trim_bam \
+--bamutils_clip_half_udg_left 2 \
+--bamutils_clip_half_udg_right 2 \
+<...>
+```
+
+To activate sex determination (using Sex.DetERRmine.py) we will provide the
+option `--run_sexdeterrmine`. Additionally, we will provide sexdeterrmine with
+the BED file of our SNPs of interest using the `--sexdeterrmine_bedfile` flag.
+Here I will use the 1240k SNP set as an example. This will cut down on
+computational time and while also providing us with an error bar around the
+relative coverage on the X and Y chromosomes.
+
+```bash
+nextflow run nf-core/eager \
+-r 2.2.0 \
+-profile sdag,shh,singularity \
+-name 'projectX_screening20200727' \
+--input 'screening20200727.tsv' \
+--fasta '../Reference/genome/hs37d5.fa' \
+--bwa_index '../Reference/genome/hs37d5/' \
+--fasta_index '../Reference/genome/hs37d5.fa.fai' \
+--seq_dict '../Reference/genome/hs37d5.dict' \
+--outdir './results/' \
+- w './work/' \
+--complexity_filter_poly_g \
+--preserve5p \
+--bwaalnn 0.01 \
+--run_trim_bam \
+--bamutils_clip_half_udg_left 2 \
+--bamutils_clip_half_udg_right 2 \
+--run_sexdeterrmine \
+--sexdeterrmine_bedfile '../Reference/genome/1240k.sites.bed' \
+<...>
+```
+
+Similarly, we will activate nuclear contamination estimation with `--run_nuclear_contamination`. This process requires us to also specify the contig name of the X chromosome in the reference genome we are using with `--contamination_chrom_name`. Here, we are using hs37d5, where the X chromosome is simply named "X".
+
+```bash
+nextflow run nf-core/eager \
+-r 2.2.0 \
+-profile sdag,shh,singularity \
+-name 'projectX_screening20200727' \
+--input 'screening20200727.tsv' \
+--fasta '../Reference/genome/hs37d5.fa' \
+--bwa_index '../Reference/genome/hs37d5/' \
+--fasta_index '../Reference/genome/hs37d5.fa.fai' \
+--seq_dict '../Reference/genome/hs37d5.dict' \
+--outdir './results/' \
+- w './work/' \
+--complexity_filter_poly_g \
+--preserve5p \
+--bwaalnn 0.01 \
+--run_trim_bam \
+--bamutils_clip_half_udg_left 2 \
+--bamutils_clip_half_udg_right 2 \
+--run_sexdeterrmine \
+--sexdeterrmine_bedfile '../Reference/genome/1240k.sites.bed' \
+--run_nuclear_contamination \
+--contamination_chrom_name 'X' \
+<...>
+```
+
+Because nuclear contamination estimates can only be provided for males, it is
+possible that we will need to get mitchondrial DNA contamination estimates for
+any females in our dataset. This cannot be done within eager (v2.2.0) and we
+will need to do this manually at a later time. However, mtDNA contamination
+estimates have been shown to only be reliable for nuclear contamination when
```

Comment:
Citation with DOI link

Reply:
for eager? We don't provide one in Introduction or Preparation. Why suddenly have one here?